src/views/LostItems/LostItemsListContainer/components/LostItemsList/LostItemsList.test.js

```diff
@@ -0,0 +1,310 @@
+import React from 'react';
+import {
+  render,
+  screen,
+} from '@testing-library/react';
+import { noop } from 'lodash';
+import { MultiColumnList } from '@folio/stripes/components';
+import {
+  FormattedDate,
+  FormattedTime,
+  FormattedMessage,
+} from 'react-intl';
+
+import '../../../../../../test/jest/__mock__';
+// eslint-disable-next-line import/no-named-as-default
+import LostItemsList, {
+  triggerOnSort,
+  basicListFormatter,
+  COLUMNS_NAME,
+  visibleColumns,
+  columnMapping,
+  columnWidths,
+} from './LostItemsList';
+import {
+  ActualCostModal,
+  ActualCostConfirmModal,
+  InstanceDetails,
+} from './components';
+import {
+  ACTUAL_COST_CONFIRM_MODAL_DEFAULT,
+  ACTUAL_COST_DEFAULT,
+  ACTUAL_COST_MODAL_DEFAULT,
+  PAGE_AMOUNT,
+} from '../../../constants';
+
+const patronName = 'name';
+
+jest.mock('./components', () => ({
+  ...jest.requireActual('./components'),
+  ActualCostModal: jest.fn(() => null),
+  ActualCostConfirmModal: jest.fn(() => null),
+  InstanceDetails: jest.fn(() => null),
+}));
+jest.mock('./util', () => ({
+  getPatronName: jest.fn(() => patronName)
```

Comment:
In test we should just check that getPatronName has been called with correct props. Could you please remove patronName?

Reply:
Done